mne/preprocessing/ica.py

```diff
@@ -786,7 +786,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```

Comment:
I also like `IC #000`, but you should make sure that the naming is consistent across mne.
BTW @agramfort managed to reference first PR to mne python :)